scripts/zip.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+rm -rf Wikipedia
+mkdir Wikipedia
+
+rm -rf dist
+
+node scripts/manifest.js > Wikipedia/manifest.webapp
+npm run Wikipedia
```

Comment:
Is this `npm run build`, or something else?

Reply:
Yes, I broke it when I renamed the directory from "build" to "Wikipedia". Should be fixed now.